### PR TITLE
Fix ABI jobs for 4.12 version

### PR DESCRIPTION
**Pull request by mhanss**
https://issues.redhat.com/browse/OCPQE-20735

### CODE REVIEW

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-vsphere-agent-compact-f28

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-vsphere-agent-compact-f28

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-vsphere-agent-compact-f28

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-agent-compact-fips-f14

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-agent-compact-fips-f14

**Comment by mhanss** [comment]
@pamoedom @jhou1 job failures are unrelated to these changes; therefore, implementing these changes should address the [bug](https://issues.redhat.com/browse/OCPQE-20735) effectively.

File: ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh [hunk 1]
```diff
@@ -85,7 +85,14 @@ pullSecret: >
...
 fi
 
-yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "${SHARED_DIR}/install-config.yaml" - <<<"
+echo "Installing from initial release $RELEASE_IMAGE_LATEST"
+oc adm release extract -a "$pull_secret_path" "$RELEASE_IMAGE_LATEST" \
+  --command=openshift-install --to=/tmp
+
+version=$(/tmp/openshift-install version | grep 'openshift-install' | awk '{print $2}' | cut -d '.' -f 1,2)
+# Add vSphere credentials if the version is 4.15 or more
+if [[ $(echo -e "4.15\n$version" | sort -V | tail -n 1) == "$version" ]]; then
```
Comment (pamoedom) [inline ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh:94]: Hi @mhanss, my 2 cents to help reduce the complexity of the condition:
~~~
version=$(/tmp/openshift-install version | grep 'openshift-install' | awk '{print $2}' | cut -d '.' -f 1,2 --output-delimiter='')
# Add vSphere credentials if the version is 4.15 or more
if [[ "${version}" -ge "415" ]]; then
~~~

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-vsphere-agent-compact-f28

File: ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh (hunk 1 above)
Comment (mhanss) [reply ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh]: Changes have been implemented.

**Pull request merged**